realtime/redisSubscriber.js

```diff
@@ -22,14 +24,20 @@ const featureToggles = require('feature-toggles');
  */
 module.exports = (io) => {
   sub.on('message', (channel, mssgStr) => {
-    // message object to be sent to the clients
-    const mssgObj = JSON.parse(mssgStr);
-    const key = Object.keys(mssgObj)[0];
+    console.log(mssgStr);
+    const mStr = Buffer.from(mssgStr, 'utf8');
+    //console.log(mStr);
+    unzipAsync(mStr)
+    .then((_mssgStr) => {
+      // message object to be sent to the clients
+      const mssgObj = JSON.parse(_mssgStr);
+      const key = Object.keys(mssgObj)[0];
 
-    /*
-     * pass on the message received through the redis subscriber to the socket
-     * io emitter to send data to the browser clients.
-     */
-    emitter(io, key, mssgObj);
+        /*
+         * pass on the message received through the redis subscriber to the socket
+         * io emitter to send data to the browser clients.
+         */
+      emitter(io, key, mssgObj);
+    });
   });
```

Comment:
I was trying to see if we can compress the samples before sending it to redis for the realtime time events.  I was able to compress it before sending it to redis but have some issues while decompressing it. I have tried a few things but could not get it working. Any other thoughts on how else we can get the decompression working. @annyhe @iamigo @harshkothari410 @pallavi2209 @jgraff2